src/cbl-mariner/1.0/helix/amd64/Dockerfile

```
@@ -0,0 +1,24 @@
+FROM cblmariner.azurecr.io/cblmariner:1.0
```

I believe the correct repo name is `base/core`.

```suggestion
FROM cblmariner.azurecr.io/base/core:1.0
```

You can see that the list of tags in `cblmariner` are old compared to `base/core`:

cblmariner:
```json
[
  "1.0",
  "1.0.20200906",
  "1.0.20201003",
  "1.0.20201223",
  "1.0.20201223-amd64",
  "1.0.20201223-arm64"
]
```

base/core:
```json
[
  "1.0",
  "1.0.20210127",
  "1.0.20210127-amd64",
  "1.0.20210127-arm64",
  "1.0.20210224",
  "1.0.20210224-amd64",
  "1.0.20210224-arm64",
  "1.0.20210331-amd64",
  "1.0.20210331-arm64",
  "1.0.20210430",
  "1.0.20210430-amd64",
  "1.0.20210430-arm64"
]
```

Can I ask where you learned of the `cblmariner` repo name?  Didn't even know about that.

---

Done.  I was asked to scout mariner in [this issue](https://github.com/dotnet/core-eng/issues/13205) and it quickly became obvious this would be a good candidate for a docker-only Helix test setup.   I found the image in [this doc](https://dev.azure.com/mariner-org/mariner/_wiki/wikis/mariner.wiki/89/Downloads)